test/repository_test.rb

```diff
@@ -68,6 +73,12 @@
             UserRepository.persist(persisted_user)
             persisted_user.created_at.must_equal persisted_user.created_at
           end
+
+          it 'touches updated_at' do
+            user.name = 'My'
```

Comment:
you do not need change any attribute here. Upon calling `persist`, the updated_at will be touched regardlessly